nifi-nar-bundles/nifi-snowflake-bundle/nifi-snowflake-services/src/main/java/org/apache/nifi/snowflake/service/StandardSnowflakeIngestManagerProviderService.java

```diff
@@ -196,7 +250,7 @@ public SimpleIngestManager getIngestManager() {
     }
 
     private AccountIdentifierFormatParameters getAccountIdentifierFormatParameters(ConfigurationContext context) {
-        final String hostUrl = context.getProperty(HOST_URL)
+        final String accountUrl = context.getProperty(ACCOUNT_URL)
```

Comment:
This code block is duplicated in `StandardSnowflakeConfigurationService` and might worth extracting it.